ouroboros-network/src/Ouroboros/Network/NodeToClient/Version.hs

```diff
@@ -40,13 +42,15 @@ nodeToClientVersionCodec = CodecCBORTerm { encodeTerm, decodeTerm }
     where
       encodeTerm NodeToClientV_1 = CBOR.TInt 1
       encodeTerm NodeToClientV_2 = CBOR.TInt (2 `setBit` nodeToClientVersionBit)
+      encodeTerm NodeToClientV_3 = CBOR.TInt (3 `setBit` nodeToClientVersionBit)
```

Comment:
@coot bors failed because I had forgotten to add this line. https://github.com/input-output-hk/ouroboros-network/issues/2215 would have prevented this mistake :slightly_smiling_face:

Reply:
I'll try to add the flags this week.

Reply:
:+1:  (It happened again. Uggh, this is annoying, I'll enable the flags locally so I can fix all errors once and for all.)